### PR TITLE
Logging refresh

### PR DESCRIPTION
**Pull request by znicholls**
Changes I'd chuck in, should pass all the tests now too

### CODE REVIEW

**Comment by znicholls** [comment]
@lewisjared

File: src/netcdf_scm/iris_cube_wrappers.py [hunk 1]
```diff
@@ -474,8 +474,12 @@ def load_data_from_identifiers(self, **kwargs):
             filepath of the file to load and the variable constraint to use.
         """
         with warnings.catch_warnings(record=True) as w:
-            # iris v2.2.0 under py3.7 raises a DeprecationWarning
-            warnings.filterwarnings("ignore", category=DeprecationWarning)
+            # iris v2.2.0 under py3.7 raises a DeprecationWarning about using collections, see
```
Comment (lewisjared) [inline src/netcdf_scm/iris_cube_wrappers.py]: Was there meant to be a link here?

**Review by lewisjared — APPROVED** [review]

File: src/netcdf_scm/iris_cube_wrappers.py (hunk 1 above)
Comment (znicholls) [reply src/netcdf_scm/iris_cube_wrappers.py]: yes oops

Comment (znicholls) [reply src/netcdf_scm/iris_cube_wrappers.py]:
```suggestion
            # iris v2.2.0 under py3.7 raises a DeprecationWarning about using collections, see https://github.com/SciTools/iris/pull/3320
```

**Comment by znicholls** [comment]
@lewisjared gtg now

**Comment by znicholls** [comment]
@lewisjared ping to merge

**Pull request merged**